src/Svg.tsx

```diff
@@ -1,38 +1,39 @@
-// @flow
-import * as React from "react"
+import * as React from "react";
 
-import uid from "./uid"
-import type { Props as HolderProps } from "./Holder"
+import { IProps } from "./Holder";
 
-export type SvgProps = {
-  children?: React.ChildrenArray<*>
-} & HolderProps
+import uid from "./uid";
+
+export interface ISvgProps extends IProps {
```

Comment:
Here all props should be required, so I don't know if there is a way to "convert" all types in required or we should create a new Interface with all props again, but required. Do you have any suggestion for that?